frontend/app/src/components/settings/general/language/LanguageSetting.vue

```diff
@@ -1,72 +1,86 @@
 <template>
-  <div class="d-flex align-center">
-    <settings-option
-      #default="{ error, success, update }"
-      class="fill-width"
-      setting="language"
-      frontend-setting
-      :error-message="t('general_settings.validation.language.error')"
-    >
-      <v-select
-        v-model="language"
-        :items="supportedLanguages"
-        item-text="label"
-        item-value="identifier"
-        outlined
-        hide-details
-        :label="t('general_settings.labels.language')"
-        persistent-hint
-        :success-messages="success"
-        :error-messages="error"
-        v-bind="rootAttrs"
-        @change="update"
+  <div>
+    <div class="d-flex align-center">
+      <settings-option
+        #default="{ error, success, update }"
+        class="fill-width"
+        setting="language"
+        frontend-setting
+        :error-message="t('general_settings.validation.language.error')"
       >
-        <template #item="{ item }">
-          <language-selector-item
-            :countries="item.countries ?? [item.identifier]"
-            :label="item.label"
-          />
-        </template>
-        <template #selection="{ item }">
-          <language-selector-item
-            :countries="item.countries ?? [item.identifier]"
-            :label="item.label"
+        <v-select
+          v-model="language"
+          :items="supportedLanguages"
+          item-text="label"
+          item-value="identifier"
+          outlined
+          hide-details
+          :label="t('general_settings.labels.language')"
+          persistent-hint
+          :success-messages="success"
+          :error-messages="error"
+          v-bind="rootAttrs"
+          @change="updateSetting($event, update)"
+        >
+          <template #item="{ item }">
+            <language-selector-item
+              :countries="item.countries ?? [item.identifier]"
+              :label="item.label"
+            />
+          </template>
+          <template #selection="{ item }">
+            <language-selector-item
+              :countries="item.countries ?? [item.identifier]"
+              :label="item.label"
+            />
+          </template>
+        </v-select>
+      </settings-option>
+      <div class="ml-2">
+        <v-tooltip open-delay="400" bottom max-width="400">
+          <template #activator="{ on }">
+            <base-external-link
+              :href="contributeUrl + '#add-a-new-language-or-translation'"
```

Comment:
do you think this should be part of the electron-interop instead? just to avoid searching for the URLs all over the place in the future

Reply:
do you mean the `contributeUrl`, or the whole `contributeUrl + '#add-a-new-language-or-translation'` ?
contributeUrl is from interop

Reply:
the whole link with `contributeUrl + '#add-a-new-language-or-translation'` it makes sense if these links are in a central place doesn't it?